src/sass/_globals.scss

```diff
@@ -0,0 +1,26 @@
+* {
+    box-sizing: border-box;
+  }
+  body {
+    $fontPrimary;
+    $textPrimary;
+  }
+  .container {
+    max-width: 480px;
+    padding: 0 15px;
+    margin: 0 auto;
+  
```

Comment:
czy są jakieś odległości stałe pomiędzy sekcjami?
jeśli tak to  można dodać 
.section {
padding: (pierwsza_wartość_odległości_w_px) 0px;
}

Reply:
miałem .section dodane z wcześniejszych projektów, ale sprawdziłem i widzę, że przy tym projekcie każda sekcja zaczyna się i kończy inaczej